napari/layers/image/image.py

```diff
@@ -297,13 +297,15 @@ def __init__(
         self._experimental_slicing_plane = SlicingPlane(
             thickness=1, enabled=False
         )
+        self._should_calc_clims = False  # calc clims on next set_view_slice?
```

Comment:
>`# calc clims on next set_view_slice?`

you tell me @tlambert03  :-p

Reply:
hah. I started with "whether to calculate..." but I _really_ wanted that to fit on one line :)

Reply:
oh lol, i hate to nitpick, but this did confuse me too. I thought it was something you were still wondering about and not a description of what was actually happening. Can we do
```suggestion
        # Whether to calculate clims on the next set_view_slice
        self._should_calc_clims = False
```